services/core-web/src/components/mine/ProjectSummaries/MineProjectSummaryTable.js

```diff
@@ -15,40 +18,70 @@ const propTypes = {
   isLoaded: PropTypes.bool.isRequired,
 };
 
-const transformRowData = (projectSummaries) =>
-  projectSummaries.map((projectSummary) => ({
-    key: projectSummary.project_summary_guid,
-    projectSummary,
-    mine_guid: projectSummary.mine_guid,
-    status_code: projectSummary.status_code,
-    documents: projectSummary.documents,
-    project_summary_id: projectSummary.project_summary_id || Strings.EMPTY_FIELD,
-    update_timestamp: formatDate(projectSummary.update_timestamp),
-  }));
+const transformRowData = (projectSummaries) => {
+  return projectSummaries.map((projectSummary) => {
+    const contact = projectSummary?.contacts?.find((c) => c.is_primary);
+
+    return {
+      key: projectSummary.project_summary_guid,
+      projectSummary,
+      mine_guid: projectSummary.mine_guid,
+      project_stage: projectSummary.status_code,
+      documents: projectSummary.documents,
+      project_summary_id: projectSummary.project_summary_id || Strings.EMPTY_FIELD,
+      project_name: projectSummary.project_summary_title,
+      project_proponent_id: projectSummary.proponent_project_id || Strings.EMPTY_FIELD,
+      project_contact: contact?.name || Strings.EMPTY_FIELD,
+      first_submitted_date: formatDate(projectSummary.create_timestamp),
+      last_updated_date: formatDate(projectSummary.update_timestamp),
```

Comment:
I chated with Arturo re: 'first_submitted_date' on his PR, I'm going to add a new date field that is set when the status goes from draft - open. I'll update this field in my PR, no action needed but something to be aware of.